test/src/specs/sync/chain_forks.rs

```diff
@@ -509,18 +507,32 @@ impl Spec for ForksContainSameTransactions {
 
         // Build `chain1`, contain the target `transaction`, with length = 61
         {
-            sleep(Duration::from_secs(2));
             node1.generate_blocks(30);
             node1.submit_transaction(&transaction);
             node1.generate_blocks(30);
         }
 
         // Build `chain2`, all the blocks are empty, with length = 51
         {
-            sleep(Duration::from_secs(2));
             node2.generate_blocks(50);
         }
 
+        let rpc_client0 = node0.rpc_client();
+        let rpc_client1 = node1.rpc_client();
+        let rpc_client2 = node2.rpc_client();
+
+        let finish_generating = wait_until(10, || {
```

Comment:
It is nonsense to check height here.
The nodes of cause are on the target height now.